src/SFA.DAS.Payments.RequiredPayments.Application.UnitTests/SFA.DAS.Payments.RequiredPayments.Application.UnitTests.csproj

```diff
@@ -1,28 +1,29 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Autofac" Version="4.8.1" />
-    <PackageReference Include="Autofac.Extras.Moq" Version="4.3.0" />
+	<PackageReference Include="Autofac" Version="7.0.1" />
+    <PackageReference Include="Autofac.Extras.Moq" Version="6.1.1" />
     <PackageReference Include="AutoFixture.NUnit3" Version="4.11.0" />
     <PackageReference Include="AutoMapper" Version="7.0.1" />
     <PackageReference Include="ESFA.DC.DateTimeProvider" Version="0.1.6-beta0001" />
-    <PackageReference Include="FluentAssertions" Version="5.6.0" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.InMemory" Version="2.2.1" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.8.0" />
+    <PackageReference Include="FluentAssertions" Version="5.9.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.InMemory" Version="7.0.20" />
```

Comment:
This version of EF Core is now out of support. We should target the latest 6.0.x version for .NET6

https://learn.microsoft.com/en-us/ef/core/what-is-new/

Reply:
Bumped down to 6.0.31